### PR TITLE
remove event bus component test

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- move eventsubscriber to its own folder tools/event-subscriber
- remove unnecessary event-bus tests

**Related issue(s)**

### CODE REVIEW

File: tests/contract/knative-channel-kafka/OWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 reviewers:
-  - event-bus-reviewers
+  - knative-eventing-kafka-reviewers
```
Comment (sayanh) [inline tests/contract/knative-channel-kafka/OWNERS:2]: Would it make sense to take out `event-bus-reviewers` and `event-bus-approvers` completely? To be specific:
- resources/nats-streaming/OWNERS
- resources/knative-provisioner-natss/OWNERS
- resources/knative-eventing/OWNERS

Comment (k15r) [reply tests/contract/knative-channel-kafka/OWNERS:2]: i wanted to consolidate all those changes in one PR

File: tools/event-subscriber/go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/kyma-project/kyma/tests/event-subscriber
```
Comment (nachtmaar) [inline tools/event-subscriber/go.mod:1]: 👍

File: tools/event-subscriber/pkg/util/logger.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httputil"
+)
+
+// Logger ...
+func Logger(h http.Handler) http.Handler {
```
Comment (nachtmaar) [inline tools/event-subscriber/pkg/util/logger.go:11]: (nit) The name of the function makes one think that a logger can be created using the function :-D 
```suggestion
func LogHandler(h http.Handler) http.Handler {
```

File: tools/event-subscriber/pkg/util/subscriber.go [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+)
+
+// Subscribers Servers API paths
+const (
+	// v1 endpoints
+	SubServer1EventsPath  = "/v1/events"
```
Comment (nachtmaar) [inline tools/event-subscriber/pkg/util/subscriber.go:14]: (nit) I can't recall that the difference between v1, v2 and v3 was. Can we have a clearer naming? Is it event-mesh or event-bus e.g.

File: tools/event-subscriber/pkg/util/subscriber.go [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+)
+
+func resultsHandlerV1(w http.ResponseWriter, _ *http.Request) {
+	mu.Lock()
+	defer mu.Unlock()
+	_ = json.NewEncoder(w).Encode(subscriberV1Result)
+}
+
+func eventsHandlerV1(_ http.ResponseWriter, r *http.Request) {
+	mu.Lock()
```
Comment (nachtmaar) [inline tools/event-subscriber/pkg/util/subscriber.go:74]: (nit) Can we move the boilerplate code to a function? I see similar code in resultHandler.* and eventsHandler.*

File: tools/event-subscriber/pkg/util/logger.go (hunk 3 above)
Comment (k15r) [reply tools/event-subscriber/pkg/util/logger.go:11]: i did not change the code... fixing the event-subscriber should be done in a later refactoring / rewriting issue

File: tools/event-subscriber/pkg/util/subscriber.go (hunk 4 above)
Comment (k15r) [reply tools/event-subscriber/pkg/util/subscriber.go:14]: again: no code was touched. only copied. Lets fix this later

File: tools/event-subscriber/pkg/util/subscriber.go (hunk 5 above)
Comment (k15r) [reply tools/event-subscriber/pkg/util/subscriber.go:74]: see previous answers.. i think this still applies here :D

**Pull request merged**